src/carousel/carousel.scss

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+@mixin iui-carousel-slider {
+  position: relative;
+  display: flex;
+  width: 100%;
+  @supports (aspect-ratio: 16 / 9) {
+    aspect-ratio: 16 / 9;
```

Comment:
When I run tests on my computer, `aspect-ratio` works, but when tested on GitHub the `aspect-ratio` doesn't work and test images fail.  How should I proceed, set a fixed width & height in the demo HTML?  I dont want to set a fixed width / height, I'd like for the user to set that as needed.

Reply:
Locally, i have same failures with tests.
I have tried to update backstopJs version (with the idea, that there is gonna be newer Chrome version), and seems that tests started to pass. So I think issue is of Chrome version, where aspect-ratio is experimental feature.
Just after udpating versino of backstop, I got over 70 failures on other components :D So not sure...
Let wait until focus PR is completed, if that could be done pretty soon ( I think there version of backstop is updated..)

Reply:
On second thought, I removed the `aspect-ratio` completely.  I'd like to suggest that 16.9 aspect ratio be the suggested default, but if needed the user should be able to override that and some users may be confused how to remove the 16.9 ratio.